In [11]:
import pandas as pd
import numpy as np

In [12]:
df1 = pd.read_excel("C:/Users/daeni/Desktop/LOVE/PJDSC2023-CarpeDatum/PJDSC2023-CarpeDatum/Dataset/INCOMING INSTAPAY TXNS_PJDSC.xlsx")

In [13]:
df1.to_csv('INCOMING INSTAPAY TXNS_PJDSC')

In [14]:
df1.head()

CUST_NUM       DATE   TIME        AMOUNT BPI_ACCT_TYPE SRC_BANK_NAME
0  5713.401964 2023-04-01  73591  12299.238264            SA         LEMUR
1  5713.401964 2023-04-05  44714   2499.424915            SA         HUMAN
2  5713.401964 2023-04-24  46302    748.744684            SA         HUMAN
3  5713.401964 2023-04-29  40612   1799.144167            SA         HUMAN
4  5713.401964 2023-04-29  24021   1499.628880            SA         HUMAN

In [15]:
df1.tail()

CUST_NUM       DATE   TIME       AMOUNT BPI_ACCT_TYPE SRC_BANK_NAME
80037  11432.368259 2023-06-30  76309  2998.982229            SA         HUMAN
80038   9868.617026 2023-06-30  38348  7499.457892            SA         HUMAN
80039   9868.617026 2023-06-30  38465    19.806571            SA         HUMAN
80040  13968.861156 2023-06-30  51968   999.240763            SA         HUMAN
80041   5523.664001 2023-06-30  65503  9999.146908            SA           DOG

In [16]:
df1.shape

(80042, 6)

**Findings**: There are 80,042 rows and 6 columns in total for the Incoming Instapay Transactions. 

In [17]:
df1.isna().head()

CUST_NUM   DATE   TIME  AMOUNT  BPI_ACCT_TYPE  SRC_BANK_NAME
0     False  False  False   False          False          False
1     False  False  False   False          False          False
2     False  False  False   False          False          False
3     False  False  False   False          False          False
4     False  False  False   False          False          False

In [18]:
df1.isna().sum()

CUST_NUM         0
DATE             0
TIME             0
AMOUNT           0
BPI_ACCT_TYPE    0
SRC_BANK_NAME    0
dtype: int64

**Findings:** There are no missing values for this dataframe. 